SplitScreenScanner.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |s|
   s.name             = 'SplitScreenScanner'
-  s.version          = '6.0.0'
+  s.version          = '6.1.0'
```

Comment:
Run `bundle exec pod install` from the example folder too, to make sure it’s on the latest version.